site/docs/v1/tech/installation-guide/_shared-state.adoc

```diff
@@ -0,0 +1,5 @@
+FusionAuth stores almost all state in its database. For clustered environments, the following interactions occur between nodes:
+
+* Cache invalidations
```

Comment:
Cache invalidations occur on timers as well as a fail safe. But if all of the "inter-node" communication is working this will allow us to synchronize cache invalidation requests. 

For logs, the API only downloads logs per node, but when you download logs from the UI we use this node listing to identify which nodes we can access and request logs from them. 

The same is true when you view logs in the UI, we will use this node listing to show you logs from each node. 

I would also add to this "future usages" as we may not document all uses since it is an internal construct.